packages/@remirror/core/src/builtins/__tests__/persistent-selection-extension.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { extensionValidityTest, renderEditor } from 'jest-remirror';
+
+import { PersistentSelectionExtension } from '..';
+
+extensionValidityTest(PersistentSelectionExtension);
+
+test('adds decoration when configured', () => {
+  const builtin = { persistentSelectionClass: 'persistent-selection' };
+  const {
+    add,
+    selectText,
+    view: { props },
+    state,
+    nodes: { p, doc },
+  } = renderEditor([], { builtin });
+
+  add(doc(p('Some text')));
+
+  selectText(4);
```

Comment:
Selection should probably be a range here. `selectText(4, 9)`. It raises a good question. Do we want the selection to even be wrapped when empty?

Reply:
@ronnyroeller, any thoughts?

Reply:
`selectText` defaults the end to the end of the text, but I can make this explicit of course.